test/versioned/elastic/elasticsearch7.tap.js

```diff
@@ -0,0 +1,401 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const test = tap.test
+const helper = require('../../lib/agent_helper')
+const params = require('../../lib/params')
+const urltils = require('../../../lib/util/urltils')
+const crypto = require('crypto')
+const DB_INDEX = `test-${randomString()}`
+const DB_INDEX_2 = `test2-${randomString()}`
+
+function randomString() {
+  return crypto.randomBytes(5).toString('hex')
+}
+
+test('Elasticsearch instrumentation', (t) => {
```

Comment:
it looks like the only differences are in some operations that in 7 need the key of body.  It may be better to just have some utility to format the request to avoid maintaining a copy of the tests with very subtle differences, thoughts?

Reply:
They're now consolidated in one file.